docs/rules/iframe-missing-sandbox.md

```diff
@@ -2,7 +2,7 @@
 
 The sandbox attribute enables an extra set of restrictions for the content in the iframe. Using sandbox attribute is considered a good security practice.
 
-See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox
+See <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox>
```

Comment:
i've never seen this markdown syntax - can you elaborate?

Reply:
Let's use the square brackets instead of the angle brackets, then

Reply:
Square brackets only work for [reference-style links](https://www.markdownguide.org/basic-syntax/#reference-style-links) when you define the URL at the bottom of the file. If we just want to include the full URL as a link, we need to use angle brackets to ensure it's treated as a link by all markdown parsers.

Reply:
I'm pretty certain they also work around a URL directly, with no reference?

Reply:
I tested it with just the square brackets only (no reference) and it didn't work before the square brackets were actually rendered.

Reply:
ah, maybe i'm just relying on it auto-rendering.

Is there any place GFM is rendered where URLs aren't auto-linkified? The only markdown parser that matters is github.com (and maybe npmjs.com).

Reply:
People often render the markdown in their own code editors during development. I use an extension in VSCode to render it which handles the URL fine either way. But I can't be sure about every other renderer someone might be using. So my recommendation is just follow the autofixable lint rule as it seems to encourage better markdown spec-compliance.